src/timesync/meson.build

```diff
@@ -1,11 +1,21 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
 systemd_timesyncd_sources = files('''
+        aes_siv.c
+        aes_siv.h
         timesyncd.c
         timesyncd-bus.c
         timesyncd-bus.h
         timesyncd-conf.c
         timesyncd-conf.h
+        timesyncd-nts-client.c
```

Comment:
As in the other PR, the new files that use openssl can be added separately, something like:

```
--- a/src/timesync/meson.build
+++ b/src/timesync/meson.build
@@ -13,6 +13,15 @@ systemd_timesyncd_sources = files('''
         timesyncd-server.h
 '''.split())
 
+if conf.get('HAVE_OPENSSL') == 1
+        systemd_timesyncd_sources += files(['src/timesync/timesyncd-nts-client.c',
+                                            'src/timesync/timesyncd-nts-client.h',
+                                            'src/timesync/timesyncd-ntske-client.c',
+                                            'src/timesync/timesyncd-ntske-client.h',
+                                            'src/timesync/timesyncd-ntske-protocol.h',
+                                            'src/timesync/timesyncd-ntske-protocol.h'])
+endif
+
 timesyncd_gperf_c = custom_target(
         'timesyncd-gperf.c',
         input : 'timesyncd-gperf.gperf',
```